discord/message.py

```diff
@@ -593,6 +597,34 @@ def to_dict(self) -> MessageReferencePayload:
     to_message_reference_dict = to_dict
 
 
+class MessageCall:
+    """Represents information about a call in a private channel.
+
+    .. versionadded:: 2.6
+    """
+
+    def __init__(self, state: ConnectionState, data: MessageCallPayload):
+        self._state: ConnectionState = state
+        self._participants: SnowflakeList = data.get("participants", [])
+        self._ended_timestamp: datetime.datetime | None = utils.parse_time(
+            data["ended_timestamp"]
+        )
+
+    @property
+    def participants(self) -> list[User | Object]:
+        """A list of :class:`User` that participated in this call.
+
+        If a user is not found in the client's cache,
+        then it will be returned as an :class:`Object`
+        """
```

Comment:
Do we have to mention this
```suggestion
        """A list of :class:`User` that participated in this call."""
```

Reply:
I included it as we have a similar warning in several other places in the library, but if it's not necessary then fair enough